### ISSUE TITLE
Display version number in issues page

### ISSUE THREAD

**Issue by sidt4** [ximion/appstream-generator#120]
There is no version information to see which version has the issues mentioned there.

https://appstream.debian.org/sid/main/issues/firefox.html
https://appstream.debian.org/sid/main/issues/firefox-esr.html

In above links, as there is an error there is no equivalent metainfo page to check for version.

**Comment by ximion** [comment]
If there is no metainfo data, then no metainfo page is generated. Issues pages are always for the latest version found in a package archive.

**Comment by sidt4** [comment]
> Issues pages are always for the latest version found in a package archive.

Yes, but displaying the version information helps in lot of places. I was asked which version of app `X` has an appstream issue. I could just paste the link, rather then explaining how to find the version information for app `X` in distro `Y`.
